stories/Select.tsx

```diff
@@ -46,16 +46,24 @@ storiesOf('Organisms|Select Components', module)
                 store.set({ selectedItem: item });
             };
 
+            const onClear = () => {
+                console.log('onClear was called');
+                store.set({ selectedItem: undefined });
+            };
+
             return (
                 <>
                     <Select<TSuggestion>
                         style={{ width: '650px' }}
                         items={SUGGESTIONS}
                         itemToString={SUGGESTION_TO_STRING}
                         selectedItem={store.get('selectedItem')}
+                        clearLabel={text('Clear button label', 'Clear')}
```

Comment:
Please add two separate stories, one for clearable select and one for Select that always have a selectedItem

Reply:
Nice point!